website/docs/reference/widgets/list.md

```diff
@@ -90,6 +82,7 @@ All of these properties are present in the property pane of the widget. The foll
 | Property                   | Description                                                                                                                                                                                                                                               |
 | -------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **Items**                  | Allows you to bind static or dynamic data collection to the widget.                                                                                                                                                                                       |
+| **Data Identifier** | Like `keys` in React, you need to select a data identifier from you data items which helps uniquely identify each item. This helps List widget  identify which items have changed, are added, or are removed. You could also combine two columns or data identifiers by enabling the JS mode. |
```

Comment:
copy should be improved, cc @jnikhila

Reply:
Have made some changes this page as part of rehaul @somangshu . let us know if the DP looks good now?